### DISCUSSION TITLE
alow to remove answered/unanswered filter when clicked again

### DISCUSSION THREAD

**Discussion by nt4f04uNd** [community/community#3652] [Discussions]
while filtering discussions by answered/unanswered, allow to click again on active filter to remove it (like with issue labels)

![image](https://user-images.githubusercontent.com/39104740/113032240-4e523900-9198-11eb-8867-84e8a94de7b3.png)

(i know that i can edit the search field)

**Comment by PranamBhat** [comment]
Hi @nt4f04uNd 

We have "Search" option in case you want to filter your search criteria more specific.

This is the default behavior of the filter's on selection of any field when we have other options to filter the content.

**Reply by nt4f04uNd** [reply]
what you are talking about is simply not true

https://user-images.githubusercontent.com/39104740/113136712-5b206c80-922c-11eb-93e8-758520ce9664.mp4

https://user-images.githubusercontent.com/39104740/113136755-65db0180-922c-11eb-9ab3-daa7d6aa1e11.mp4

**Comment by PranamBhat** [comment]
Why would you click same filter multiple times to expect different behavior. It's not the behavior meant for filter when we have enough search filter categories!!

**Reply by nt4f04uNd** [reply]
it's inconsistent

**Comment by romandev0318** [comment]
I think we need to add answer/unanswered option to search filter. Thanks!
